.travis.yml

```diff
@@ -14,5 +15,9 @@ install:
 script:
 - npm run build
 - npm run test:coverage -- --runInBand
+- npm run start &
+- cypress install
```

Comment:
I would think that the `cypress install` should be done as part of the `install` step.